apps/examples/testcase/Kconfig

```diff
@@ -32,6 +32,7 @@ source "$APPSDIR/examples/testcase/ta_tc/wifi_manager/itc/Kconfig"
 source "$APPSDIR/examples/testcase/ta_tc/mqtt/utc/Kconfig"
 source "$APPSDIR/examples/testcase/ta_tc/mqtt/itc/Kconfig"
 source "$APPSDIR/examples/testcase/ta_tc/media/utc/Kconfig"
+source "$EXTERNALDIR/libcxx-test/Kconfig"
```

Comment:
Do we need to call this from here?
For external, we don't need to follow our testcase. Testcase is for our os and framework.